client/src/redux/sagas/jobPostingSaga.js

```diff
@@ -51,6 +53,14 @@ function* removeStageInJobPosting({ payload }) {
   yield put(actions.removeStageInJobPosting(payload.stage))
 }
 
+function* addStartDate({ payload }) {
```

Comment:
Näitä ei tarvitse määritellä saagoina, kun ei ole mitään sivuvaikutuksia, vaan ainoastaan kutsutaan action:eitä